Aspen-Discovery/aspen-discovery
# Remove False Required Flag

Remove 'required' flag for Order Record Status to treat as Under Consideration in IndexingProfile.php
Updated release notes

To test:
Go to Indexing Profiles and check if Order Record Status to treat as Under Consideration is required or not (it should not be) - This setting only affects Sierra
